utils/find-overlay-dependencies.sh

```diff
@@ -14,39 +14,68 @@
 # This script uses zsh for its associative array support, because it is only
 # intended to be run on macOS and macOS bash is very old.
 
-if [[ $# -ne 1 ]]; then
-  echo 'usage:' $0 '<module-name>' >&2
+function usage() {
+  echo 'usage:' $0 '<module-name> [update]' >&2
   exit 1
-fi
+}
+
+UPDATE_CMAKE=0
+case $# in
+1) ;;
+2) if [[ $2 != 'update' ]]; then
+    usage
+  fi
+  UPDATE_CMAKE=1 ;;
+*)
+  usage ;;
+esac
 
 OVERLAYS_PATH=$(dirname "$0")/../stdlib/public/SDK/
```

Comment:
This should probably be consistent with the `-loop` script in using `$PWD` or not.